pydomcfg/domzgr/zco.py

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env python
+
+"""
+Class to generate NEMO v4.0 standard geopotential z-coordinates
+"""
+
+
+import numpy as np
+from xarray import Dataset
+
+from .zgr import Zgr
+
+
+class Zco(Zgr):
+
+    # In future we may want to get rid of this
+    # and use a more pythonic way. We could do
+    # it when  we read the namelist.
+    pp_to_be_computed = 999999.0
+
+    """
+      Class to generate geopotential z-coordinates dataset objects.
```

Comment:
Probably darglint doesn't like the indentation: Class should start under the first `"`